src/core/lib/experiments/rollouts.yaml

```diff
@@ -87,7 +87,7 @@
 - name: pick_first_happy_eyeballs
   default: true
 - name: promise_based_client_call
-  default: broken
+  default: false
```

Comment:
Doesn't this mean that the experiment will still be enabled in end2end tests? It looks like it based on the generated `experiments.bzl`. This experiment will still be blocking PR submission.

Reply:
Oh absolutely. There's a very strong need for this to be enabled for some work loads very soon.